speed-dreams/source-2.2.3/src/modules/sound/snddefault/OpenalSoundInterface.cpp

```diff
@@ -213,6 +214,33 @@ Sound* OpenalSoundInterface::addSample (const char* filename, int flags, bool lo
 	sound_list.push_back(sound);
 	return sound;
 }
+
+// SIMULATED DRIVING ASSISTANCE: Update intervention sounds
+void updateInterventionSounds(CarSoundData** car_sound_data, std::unordered_map<InterventionAction, Sound*> intervention_sounds) {
```

Comment:
p_?

Reply:
and shouldn't this be commented in our coding style as well

Reply:
Update to follow code standard
```suggestion
void UpdateInterventionSounds(CarSoundData** p_carSoundData, std::unordered_map<InterventionAction, Sound*> p_interventionSounds)
{
```